src/Sarif.FunctionalTests/SarifConverterTestData/FxCop/FxCopExampleReport2.xml.sarif

```diff
@@ -5,7 +5,22 @@
       "toolInfo": {
         "name": "FxCop"
       },
-      "runInfo": {},
+      "runInfo": {
+        "fileInfo": {
+          "d:\\SecDevTools\\Main\\bin\\x86\\Debug\\FxCop\\FxCopTestData\\Crypto\\SecurityCryptographyRuleTests.dll": [
```

Comment:
so your key here is a file path not a URI. was this intentional? looks like this is actually a bug in the fxcop converter. note that if we take my suggestion and fix the JSON serializer for this case, this would automatically get resolved.